### PR TITLE
Enabled Ninja build

### PR DESCRIPTION
**Pull request by mryzhov**

### CODE REVIEW

**Review by apaniukov — APPROVED** [review]

**Comment by Wovchena** [comment]
It didn't help 
```
CMake Error:
  Running

   'C:/Users/vzlobin/r/ninja.exe' '-C' 'C:/Users/vzlobin/r/g/build/CMakeFiles/CMakeScratch/TryCompile-aqlo4u' '-t' 'recompact'

  failed with:

   ninja: error: build.ninja:53: bad $-escape (literal $ must be written as $$)





CMake Error at C:/Program Files/CMake/share/cmake-3.26/Modules/CMakeDetermineCompilerABI.cmake:57 (try_compile):
  Failed to generate test project build system.
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.26/Modules/CMakeTestCCompiler.cmake:26 (CMAKE_DETERMINE_COMPILER_ABI)
  build/_deps/sentencepiece-src/CMakeLists.txt:23 (project)
```

**Review by ilya-lavrenov — APPROVED** [review]
Should we explicitly set Ninja as generator in some builds on GHA?

**Comment by mryzhov** [comment]
> Should we explicitly set Ninja as generator in some builds on GHA?

yes, will add in this pr

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -243,6 +244,16 @@ jobs:
...
 
+      - name: Install build dependencies
+        run: |
+          Invoke-WebRequest https://github.com/ninja-build/ninja/releases/download/v1.11.1/ninja-win.zip -OutFile ninja-win.zip -MaximumRetryCount 10
+          Expand-Archive -Force ninja-win.zip
+          # Add it to the GitHub Path so it would be available in the subsequent steps
+          Add-Content -Path $env:GITHUB_PATH -Value "${{ github.workspace }}/ninja-win"
+
+      - name: Configure Developer Command Prompt for Microsoft Visual C++
+        uses: ilammy/msvc-dev-cmd@v1
```
Comment (ilya-lavrenov) [inline .github/workflows/windows.yml:255]: what toolchain is used by default? maybe we need to hardcode it explicitly?

Comment (mryzhov) [reply .github/workflows/windows.yml:255]: the latest available 14.41

**Pull request merged**